agreementbot/api.go

```diff
@@ -387,6 +388,9 @@ func (a *API) agreement(w http.ResponseWriter, r *http.Request) {
 				} else {
 
 					for _, agreement := range ags {
+						if err := obscureSecretDetails(&agreement); err != nil {
+							glog.Warning(APIlogString(fmt.Sprintf("failed to obscure secret details, error: %v", err)))
```

Comment:
You missed the /agreement/{id} section right before this code, it needs to be updated too.